google-cloud-logging/src/test/java/com/google/cloud/logging/LoggingImplTest.java

```diff
@@ -173,6 +178,40 @@ public LogMetric apply(MetricInfo metricInfo) {
   private LoggingRpc loggingRpcMock;
   private Logging logging;
 
+  private void configureListLogsTests(List<String> returnedList, String cursor) {
```

Comment:
to make it easy for folks not familiar with the codebase - add a comment explaining what is expected in this configuration as the end result

Reply:
It is just auxiliary method. I will do more refactoring on the test code latter to generalize these methods. I will comment them after that work is done.